meta-xt-prod-cockpit-rcar-driver-domain/recipes-extended/vis-service/files/VisClient.cpp

```diff
@@ -19,72 +19,43 @@
 #include "VisClient.h"
 #include <limits>
 #include <vector>
+#include <linux/r_taurus_cluster_protocol.h>
```

Comment:
Do we need to add this header to PR? Or it is used from another place?

Reply:
Ofcourse we need the CLUSTER_SPEED and all the rest CLUSTER_... variables